src/common/nodelist-utils/index.js

```diff
@@ -1,6 +1,6 @@
 // returns index of first node in NodeList with matching first character (case-insenstive)
 function findNodeWithFirstChar(nodeList, char) {
-    return [...nodeList].findIndex((el) => el.innerText.charAt(0).toLowerCase() === char);
+    return [...nodeList].findIndex((el) => el.innerText.charAt(0).toLowerCase() === char.toLowerCase());
```

Comment:
Just a quick question, is there a possibility either el.innerText or char will be null? It could cause an error if it does.

Reply:
If called purely as a utility, then yes. From menu component as it stands, I don't believe so.